stdlib/bytes.mli

```diff
@@ -126,7 +124,7 @@ val extend : bytes -> int -> int -> bytes
     the corresponding side of [s].
     @raise Invalid_argument if the result length is negative or
     longer than {!Sys.max_string_length} bytes.
-    @since 4.05.0 *)
+    @since 4.05.0 in [BytesLabels] *)
```

Comment:
Good call. It does not. {!BytesLabels} doesn't work either. I have replaced with "in labeled module" for now, pending a decision on the bigger @ since issue.